src/Perspex.Controls/Expander.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Perspex.Animation;
+using Perspex.Controls.Primitives;
+
+namespace Perspex.Controls
+{
+    public enum ExpandDirection
+    {
+        Down,
+        Up,
+        Left,
+        Right
+    }
+
+    public class Expander : HeaderedContentControl
+    {
+        static Expander()
+        {
+            PseudoClass(ExpandDirectionProperty, d => d == ExpandDirection.Down, ":ExpandDirectionDown");
```

Comment:
Ok, that's fine then, though it might be worth being consistent in using pseudoclasses or property selectors.